lib/airbrake/rack/notice_builder.rb

```diff
@@ -39,64 +43,67 @@ def initialize(rack_env)
       # @return [Airbrake::Notice] the notice with extra information
       def build_notice(exception)
         notice = Airbrake.build_notice(exception)
-
-        add_context(notice)
-        add_session(notice)
-        add_params(notice)
-        add_environment(notice)
-
+        NoticeBuilder.builders.each { |filter| filter.call(notice, @request) }
         notice
       end
 
-      private
-
-      def add_context(notice)
+      add_builder do |notice, request|
```

Comment:
Could you add small comments explaining what a builder adds? So people don't have to read code to understand what it attaches?

For example:

```
# Adds context.
add_builder do |notice, request|
  ...
```